src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -970,7 +985,7 @@ boolean isBinary() {
      * @return true if the JDBC type is textual
      */
     private final static EnumSet<Category> textualCategories = EnumSet.of(Category.CHARACTER, Category.LONG_CHARACTER,
-            Category.CLOB, Category.NCHARACTER, Category.LONG_NCHARACTER, Category.NCLOB);
+    Category.CLOB, Category.NCHARACTER, Category.LONG_NCHARACTER, Category.NCLOB, Category.JSON); //FIXME: JSON is textual?
```

Comment:
you have a //fixme comment. Was this resolved? how does textualCategories impact APIs ?